deface/__init__.py

```diff
@@ -1 +0,0 @@
-from ._version import __version__
```

Comment:
Probably not a good idea to remove this.
Unless I'm mistaken, that's how the upstream code does releases.
If that blocks importing through github, we need to find a way to support both.